src/phoneField/index.tsx

```diff
@@ -180,39 +194,44 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
       textFieldPlaceholder,
       defaultPhoneValue,
       isInline,
+      error,
     } = this.props
 
     const baseClassName = cc([prefix({ phoneField: true })])
+    const wrapperClassName = `${baseClassName}-wrapper`
 
     const classNames = cc([
-      isInline ? `${baseClassName}--inline` : baseClassName,
+      isInline ? `${wrapperClassName}--inline` : wrapperClassName,
       this.props.className,
-      this.state.hasFocus && `${baseClassName}--hasFocus`,
+      this.state.hasFocus && `${wrapperClassName}--hasFocus`,
     ])
 
     return (
-      <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
-        <SelectField
-          name={FIELDS.PHONEREGION}
-          options={this.state.countryData}
-          ariaLabel={selectFieldLabel}
-          defaultValue={this.props.defaultRegionValue}
-          onChange={this.handleChange}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
-        <TextField
-          type="tel"
-          name={FIELDS.PHONENUMBER}
-          placeholder={textFieldPlaceholder}
-          defaultValue={defaultPhoneValue}
-          onChange={this.handleChange}
-          title={textFieldTitle}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
+      <div className={cc([baseClassName, prefix({ error: !!error })])}>
+        <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
```

Comment:
I'm not sure how this aria label will work in screen readers, we will have bropwse focus on this container but most likely on a nested focusable inputs inside.

Reply:
My gut feeling will be to just remove it.

Reply:
Aria labels don't impact focus, it just provides additional informations when the user focus/interacts with the item. I'm more concerned about putting it on a `<div />`, since it can't be interacted with, I'm afraid the aria attribute won't have any effect =/